packages/ComponentLibrary/src/components/Nav/Nav.icons.Mock.ts

```diff
@@ -0,0 +1,14 @@
+import HomeIcon from '@mui/icons-material/Home';
+import Settings from '@mui/icons-material/Settings';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import AppsIcon from '@mui/icons-material/Apps';
+
+export const iconActions = [
```

Comment:
Leave here a comment to know that in a near future, it must be a param at rendering in the metadata